### ISSUE TITLE
BERTopic Octis

### ISSUE THREAD

**Issue by AriG05** [MaartenGr/BERTopic#416]
Hi Maarten,

first of all thank you very much for BERTopic! It is really wonderful to work with it.
In some other threads I saw you recommending Ocits as a package to implement evaluation measures. 
As I'm quite new to Python I wanted to ask if you had any suggestions on how to implement BERTopic as a external model within Ocits (overwriting their class) to be able to use their Optimizer etc.
Thank you very much in advance!
Best,
Ariane

**Comment by MaartenGr** [comment]
Glad to hear that you enjoy the package! 

I have not used BERTopic yet for overwriting their class as I am in general not a huge fan of optimizing BERTopic without human evaluation in between. There might be a chance that you overfit on a metric whilst using any optimizer. However, you can follow along with [this](https://colab.research.google.com/github/MIND-Lab/OCTIS/blob/master/examples/OCTIS_LDA_training_only.ipynb) notebook to understand what the `output` variable needs to look like before using it in something like: 

```python
from octis.evaluation_metrics.coherence_metrics import Coherence

npmi = Coherence(texts=dataset.get_corpus(), topk=10, measure='c_npmi')
npmi_score = npmi.score(output)
```

By focusing on the `output` variable, there is no need for overwriting their modelclass yet.

**Comment by AriG05** [comment]
Thank you very much for the swift reply!
I can see why that could be a problem. I will go from here and try an iterative procedure to tune my hyperparameters.
